public/app/plugins/panel/table/transformers.ts

```diff
@@ -73,6 +73,56 @@ transformers['timeseries_to_columns'] = {
   }
 };
 
+transformers['grouping_to_rows_columns'] = {
+  description: 'Grouping to rows and columns',
+  getColumns: function() {
+    return [];
+  },
+  transform: function(data, panel, model) {
+    // Getting the columns
+    const points = new Map<string, Map<string, any>>();
+
+    const columns = new Set<string>();
+    const rows = new Set<string>();
+
+    for (let series of data) {
+      const columnsString = series.target.slice(
+        series.target.indexOf("{") + 1, -1);
+      const columnsArray = columnsString.split(", ");
+      const column = columnsArray[0].split(": ")[1];
+      const row = columnsArray[1].split(": ")[1];
+      columns.add(column);
+      rows.add(row);
+
+      if (!points.has(column)) {
+        points.set(column, new Map<string, any>());
+      }
+
+      // We take only the first measure reported (datapoints[0]) and
+      // We don't care about the timestamp of that measure (datapoints[0][0])
+      points.get(column).set(row, series.datapoints[0][0]);
```

Comment:
I am taking only the first data point, should I apply an average function to all the data points at this moment instead?